priv/repo/migrations/20230104213552_grant_admin_delete_on_projects.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Ret.Repo.Migrations.GrantAdminDeleteOnProjects do
+  use Ecto.Migration
+
+  def change do
+    execute "GRANT DELETE ON ret0_admin.projects TO ret_admin",
+            "REVOKE DELETE ON ret0_admin.projects TO ret_admin"
```

Comment:
Why do we grant ret_admin privileges and then immediately revoke?

Reply:
Ah, I see how it could look that way! Unlike `execute/1`, `execute/2` allows us to create a reversible SQL call. The `change/0` definition above is equivalent to:
```elixir
def up do
  execute "GRANT DELETE ON ret0_admin.projects TO ret_admin"
end

def down do
  execute "REVOKE DELETE ON ret0_admin.projects TO ret_admin"
end
```

Reply:
Ah thank you very much!